app/controller/command/commands/team.py

```diff
@@ -442,7 +451,34 @@ def remove_helper(self, param_list, user_id) -> ResponseTuple:
             self.gh.remove_team_member(user.github_username,
                                        team.github_team_id)
             self.facade.store(team)
-            msg = "Removed User from " + param_list['team_name']
+
+            msg = "Removed User from " + command_team
+
+            # If this team is a team with special permissions, demote the user
+            demoted_level = None
+            if command_team == self.config.github_team_leads:
+                # Leads can only be demoted to member
+                demoted_level = Permissions.member
```

Comment:
Similar to above. If the admin is also a team lead, they will be demoted to member if they are removed from the team leads team.

Reply:
fe49d42f937104ba2b244ac6a67bb58bc69929b0